src/course-home/mentoring-tab/MentoringTab.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import ChatWindow from './ChatWindow';
+
+import messages from './messages';
+import './MentoringTab.scss';
+
+const MentoringTab = ({ intl }) => (
```

Comment:
[optional]: Let's add a little support message about what AI Mentoring is

Reply:
As an additional support option, we can add a paragon product tour to quickly introduce the user to the mentoring application. What do you think?